addons/purchase/migrations/8.0.1.1/post-migration.py

```diff
@@ -139,7 +163,7 @@ def migrate_procurement_order(cr):
              AND pol.{move_dest_id} IS NOT NULL
              AND pol.{move_dest_id} = proc.move_dest_id
         """.format(
-            purchase_id=openupgrade.get_legacy_name('purchase_id'),
+            purchase_id=legacy_purchase_id,
```

Comment:
`sql.Identifier`?

Reply:
Not, because this is formatting a `str`. I guess this is wrong, but I didn't want to change everything.